analytics/terraform/spark-k8s-operator/examples/benchmark/tpcds-benchmark-1t.yaml

```diff
@@ -81,7 +81,7 @@ spec:
   executor:
     volumeMounts:
       - name: spark-local-dir-1
-        mountPath: /data1
+        mountPath: /mnt/k8s-disks
```

Comment:
Did you test any of these examples after the new RAID0 config? Pod `mountPath` (e.g., /data1) is new directory and its different form the mountPoint(`/mnt/k8s-disks`).

Reply:
reverted the `/data1` changes. I did test the `pyspark-pi-job.yaml`: 
```
k get pods -n spark-team-a -w
NAME                          READY   STATUS    RESTARTS   AGE
pyspark-pi-karpenter-driver   0/1     Pending   0          0s
pyspark-pi-karpenter-driver   0/1     Pending   0          3s
pyspark-pi-karpenter-driver   0/1     Pending   0          47s
pyspark-pi-karpenter-driver   0/1     ContainerCreating   0          48s
pyspark-pi-karpenter-driver   1/1     Running             0          3m11s
pythonpi-8fbf89894a5269c5-exec-1   0/1     Pending             0          0s
pythonpi-8fbf89894a5269c5-exec-2   0/1     Pending             0          0s
pythonpi-8fbf89894a5269c5-exec-1   0/1     Pending             0          1s
pythonpi-8fbf89894a5269c5-exec-2   0/1     Pending             0          1s
pythonpi-8fbf89894a5269c5-exec-1   0/1     Pending             0          41s
pythonpi-8fbf89894a5269c5-exec-2   0/1     Pending             0          41s
pythonpi-8fbf89894a5269c5-exec-1   0/1     ContainerCreating   0          42s
pythonpi-8fbf89894a5269c5-exec-2   0/1     ContainerCreating   0          42s
pythonpi-8fbf89894a5269c5-exec-1   1/1     Running             0          112s
pythonpi-8fbf89894a5269c5-exec-2   1/1     Running             0          112s
pythonpi-8fbf89894a5269c5-exec-1   1/1     Terminating         0          118s
pythonpi-8fbf89894a5269c5-exec-2   1/1     Terminating         0          118s
pyspark-pi-karpenter-driver        0/1     Completed           0          5m20s
pyspark-pi-karpenter-driver        0/1     Completed           0          5m22s
pythonpi-8fbf89894a5269c5-exec-2   0/1     Terminating         0          2m5s
pythonpi-8fbf89894a5269c5-exec-2   0/1     Terminating         0          2m5s
pythonpi-8fbf89894a5269c5-exec-2   0/1     Terminating         0          2m5s
pythonpi-8fbf89894a5269c5-exec-1   0/1     Terminating         0          2m5s
pythonpi-8fbf89894a5269c5-exec-1   0/1     Terminating         0          2m5s
pythonpi-8fbf89894a5269c5-exec-1   0/1     Terminating         0          2m5s
pyspark-pi-karpenter-driver        0/1     Terminating         0          8m6s
pyspark-pi-karpenter-driver        0/1     Terminating         0          8m6s
```

Reply:
And the `./taxi-trip-execute.sh` example:

```
k get pods -n spark-team-a -w
NAME        READY   STATUS    RESTARTS   AGE
taxi-trip   0/1     Pending   0          7s
taxi-trip   0/1     Pending   0          44s
taxi-trip   0/1     Init:0/1   0          45s
taxi-trip   0/1     Init:0/1   0          55s
taxi-trip   0/1     PodInitializing   0          57s
taxi-trip   1/1     Running           0          84s
taxi-trip-exec-1   0/1     Pending           0          0s
taxi-trip-exec-2   0/1     Pending           0          0s
taxi-trip-exec-3   0/1     Pending           0          0s
taxi-trip-exec-1   0/1     Pending           0          0s
taxi-trip-exec-4   0/1     Pending           0          0s
taxi-trip-exec-2   0/1     Pending           0          1s
taxi-trip-exec-3   0/1     Pending           0          1s
taxi-trip-exec-4   0/1     Pending           0          1s
taxi-trip-exec-3   0/1     Pending           0          46s
taxi-trip-exec-2   0/1     Pending           0          46s
taxi-trip-exec-1   0/1     Pending           0          46s
taxi-trip-exec-3   0/1     Init:0/1          0          47s
taxi-trip-exec-1   0/1     Init:0/1          0          47s
taxi-trip-exec-2   0/1     Init:0/1          0          47s
taxi-trip-exec-4   0/1     Pending           0          52s
taxi-trip-exec-4   0/1     Init:0/1          0          52s
taxi-trip-exec-4   0/1     Init:0/1          0          53s
taxi-trip-exec-4   0/1     PodInitializing   0          54s
taxi-trip-exec-1   0/1     Init:0/1          0          56s
taxi-trip-exec-2   0/1     Init:0/1          0          56s
taxi-trip-exec-3   0/1     Init:0/1          0          57s
taxi-trip-exec-1   0/1     PodInitializing   0          57s
taxi-trip-exec-2   0/1     PodInitializing   0          57s
taxi-trip-exec-3   0/1     PodInitializing   0          58s
taxi-trip-exec-1   1/1     Running           0          77s
taxi-trip-exec-2   1/1     Running           0          77s
taxi-trip-exec-3   1/1     Running           0          77s
taxi-trip-exec-4   1/1     Running           0          81s
taxi-trip-exec-1   1/1     Terminating       0          11m
taxi-trip-exec-2   1/1     Terminating       0          11m
taxi-trip-exec-3   1/1     Terminating       0          11m
taxi-trip-exec-4   1/1     Terminating       0          11m
taxi-trip-exec-4   0/1     Terminating       0          11m
taxi-trip-exec-4   0/1     Terminating       0          11m
taxi-trip-exec-4   0/1     Terminating       0          11m
taxi-trip-exec-2   0/1     Terminating       0          11m
taxi-trip-exec-2   0/1     Terminating       0          11m
taxi-trip-exec-2   0/1     Terminating       0          11m
taxi-trip-exec-3   0/1     Terminating       0          11m
taxi-trip-exec-3   0/1     Terminating       0          11m
taxi-trip-exec-3   0/1     Terminating       0          11m
taxi-trip-exec-1   0/1     Terminating       0          11m
taxi-trip-exec-1   0/1     Terminating       0          11m
taxi-trip-exec-1   0/1     Terminating       0          11m
taxi-trip          0/1     Completed         0          13m
taxi-trip          0/1     Completed         0          13m
taxi-trip          0/1     Terminating       0          16m
taxi-trip          0/1     Terminating       0          16m
```

<img width="730" alt="image" src="https://github.com/awslabs/data-on-eks/assets/10913471/d5deb0e6-cb98-48a4-9b84-003e939a97c8">